x-pack/plugins/fleet/server/services/epm/package_service.ts

```diff
@@ -222,9 +235,9 @@ class PackageClientImpl implements PackageClient {
     return installedTransforms;
   }
 
-  #runPreflight() {
+  async #runPreflight(requiredAuthz?: FleetAuthzRouteConfig['fleetAuthz']) {
     if (this.preflightCheck) {
-      return this.preflightCheck();
+      return await this.preflightCheck(requiredAuthz);
```

Comment:
nit: but probably no need to `await` here
```suggestion
      return this.preflightCheck(requiredAuthz);
```

Reply:
I made `preflighCheck` async [here](https://github.com/elastic/kibana/pull/163727/files#diff-36216259ff01a488fce1217bfc2455c3725ba8f87ad647c7541ea7498b7ed9a5), that's why I added `await`.